src/test/java/core/basesyntax/strategy/transaction/BalanceHandlerTest.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax.strategy.transaction;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.dao.TransactionDaoImpl;
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.Fruit;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceHandlerTest {
+    private static TransactionHandler handler;
+    private static Fruit fruit;
+    private static String fruitType = "banana";
+    private static int quantity = 20;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handler = new BalanceHandler(new TransactionDaoImpl());
+        fruit = new Fruit(fruitType, quantity);
+    }
+
+    @Test
+    public void doBalanceTransaction_Fruit_Ok() {
```

Comment:
Why do you use Fruit in name of test? You can also avoid 'Balance' in name of test - we know it from name of the class. Use original method's name. Fix it in other handlers
```suggestion
    public void doTransaction_ok() {
```